<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>w_m_member_paramset_Dialog</class>
 <widget class="QDialog" name="w_m_member_paramset_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>772</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>会员设置</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0" colspan="2">
          <widget class="QLabel" name="label_5">
           <property name="styleSheet">
            <string notr="true">font-weight:bold;</string>
           </property>
           <property name="text">
            <string>卡种类设置</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="lds_checkbox_wrap" name="checkBox_discount">
           <property name="text">
            <string>优惠卡(仅积分和折扣使用)</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="lds_checkbox_wrap" name="checkBox_save">
           <property name="text">
            <string>充值卡(储值卡，预充值消费)</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="2">
          <widget class="QLabel" name="label">
           <property name="styleSheet">
            <string notr="true">color:#ff650b;</string>
           </property>
           <property name="text">
            <string>注:充值卡具有积分和折扣的功能</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QFrame" name="frame_2">
         <layout class="QGridLayout" name="gridLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <item row="0" column="0" colspan="2">
           <widget class="QLabel" name="label_6">
            <property name="styleSheet">
             <string notr="true">font-weight:bold;</string>
            </property>
            <property name="text">
             <string>充值</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>最大充值金额</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_max_deposit">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item row="2" column="0">
           <widget class="lds_checkbox_wrap" name="checkBox_m_deposit_negative">
            <property name="text">
             <string>充值与实收金额必须相等</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="lds_checkbox_wrap" name="checkBox_m_card_paydiscount">
            <property name="text">
             <string>充值卡会员付款时自动折扣</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="2">
           <widget class="lds_checkbox_wrap" name="checkBox_deposit_pay_all">
            <property name="text">
             <string>充值卡付款必须全单付满</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0" colspan="2">
          <widget class="QLabel" name="label_7">
           <property name="styleSheet">
            <string notr="true">font-weight:bold;</string>
           </property>
           <property name="text">
            <string>其他</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="lds_checkbox_wrap" name="checkBox_discount_view">
           <property name="text">
            <string>折扣时显示会员折扣详情</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="lds_checkbox_wrap" name="checkBox_local_select">
           <property name="text">
            <string>会员消费时可查找会员卡</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="lds_checkbox_wrap" name="checkBox_member_bank">
           <property name="text">
            <string>会员作废的时候打印账单</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="lds_checkbox_wrap" name="checkBox_m_fetch_base">
           <property name="text">
            <string>取款可取保本金额</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0" colspan="2">
          <widget class="lds_checkbox_wrap" name="checkBox_member_pay">
           <property name="text">
            <string>充值会员付款允许负数</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>卡设置</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>卡类型</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="lds_ComboBox" name="comboBox_vip_card_type"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_m_clearAll">
           <property name="text">
            <string>卡类型重置</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_cika">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="lds_checkbox_wrap" name="checkBox_input_memberno">
              <property name="text">
               <string>可手动输入会员号</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_ic">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="1" column="0">
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>读写器端口</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="lds_ComboBox" name="comboBox_vip_ic_port"/>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>IC卡读写器</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_icpwd">
              <property name="text">
               <string>IC卡读写密码</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="lds_ComboBox" name="comboBox_vip_ic_machine"/>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="lineEdit_vip_ic_password">
              <property name="text">
               <string>FFFFFF</string>
              </property>
              <property name="echoMode">
               <enum>QLineEdit::Normal</enum>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QPushButton" name="pushButton_ic_pwd_def">
              <property name="text">
               <string>默认</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QTextEdit" name="textEdit">
         <property name="styleSheet">
          <string notr="true">color:#ff650b;
background:transparent;</string>
         </property>
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Ubuntu'; font-size:11pt;&quot;&gt;IC卡:	可存储少量信息&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Ubuntu'; font-size:11pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Ubuntu'; font-size:11pt;&quot;&gt;磁卡:	随卡提供一个编号，资料存于数据库，连锁模式使用时需要建立总部连接，能自动提供编号的ID卡或条码也可以做磁卡使用&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Ubuntu'; font-size:11pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Ubuntu'; font-size:11pt;&quot;&gt;明卡：	仅用于单店，可以是任意打印或印刷形式卡片，使用时录入查询或选择会员&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="lds_toolbar" name="widget"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>lds_ComboBox</class>
   <extends>QComboBox</extends>
   <header>lds_combobox.h</header>
  </customwidget>
  <customwidget>
   <class>lds_toolbar</class>
   <extends>QFrame</extends>
   <header>lds_toolbar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>lds_checkbox_wrap</class>
   <extends>QCheckBox</extends>
   <header>lds_checkbox_wrap.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
